priv/repo/migrations/20220701115243_create_peers_metrics.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Repo.Migrations.CreatePeersMetrics do
+  use Ecto.Migration
+
+  @chunk_time_interval Application.get_env(:membrane_rtc_engine_timescaledb, Repo)[:chunk_time_interval] || "10 minutes"
+
+  def change do
+    create table(:peers_metrics, primary_key: false) do
+      add(:time, :naive_datetime_usec, null: false)
+      add(:peer_id, :string, null: false)
+      add(:"peer.metadata", :string)
+      add(:"ice.binding_requests_received", :integer)
+      add(:"ice.binding_responses_sent", :integer)
+      add(:"ice.bytes_received", :integer)
+      add(:"ice.bytes_sent", :integer)
+      add(:"ice.packets_received", :integer)
+      add(:"ice.packets_sent", :integer)
+    end
+
+    execute("SELECT create_hypertable('peers_metrics', 'time', chunk_time_interval => INTERVAL '#{@chunk_time_interval}')")
```

Comment:
`down` is missing, this execute won't be reversed. Ensure all the migration can be reverted (`mix ecto.rollback` will work on them)